src/views/Conference.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="conference-page"
+    >
+      <div
+        id="jitsi"
+      />
+    </div>
+  </Page>
+</template>
+<script>
+// eslint-disable-next-line import/no-extraneous-dependencies
+import JitsiMeetExternalAPI from 'jitsi-iframe-api';
+import Page from '../components/layout/Page.vue';
+import { IS_MOBILE_DEVICE } from '../utils/util';
+
+export default {
+  name: 'Conference',
+  components: {
+    Page,
+  },
+  mounted() {
+    // eslint-disable-next-line no-new
+    new JitsiMeetExternalAPI('test.league.aeternity.org', {
+      parentNode: document.querySelector('#jitsi'),
+      width: '100%',
+      height: 440,
+      roomName: this.$route.params.room,
+      configOverwrite: {
+        disableDeepLinking: IS_MOBILE_DEVICE,
```

Comment:
```suggestion
        disableDeepLinking: !IS_MOBILE_DEVICE,
```
Actually, deep linking should be enabled on mobile. But still, this is not accurate, because we should track sdk on desktop.

Reply:
We need to disable it to make it work from mobile browser and disable the an app linking.